mica-core/src/main/java/org/obiba/mica/core/domain/NetworkTable.java

```diff
@@ -1,44 +0,0 @@
-/*
- * Copyright (c) 2017 OBiBa. All rights reserved.
- *
- * This program and the accompanying materials
- * are made available under the terms of the GNU Public License v3.0.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
-
-package org.obiba.mica.core.domain;
-
-import java.io.Serializable;
-
-import com.google.common.base.MoreObjects;
-
-/**
- * Represents a opal {@link OpalTable} that is associated to a {@link org.obiba.mica.network.domain.Network}.
- */
-public class NetworkTable extends OpalTable implements Serializable {
```

Comment:
I am not sure I understand what you are saying. I just would like to make sure that legacy harmo datasets are deserialized without errors. The networkTable field could exist without being used.